src/lib/base-importer.ts

```diff
@@ -0,0 +1,18 @@
+import { bases } from 'multiformats/basics'
+
+export default async function baseImporter<T extends string> (prefix: T) {
+  console.log(`bases: `, bases);
+  for (const base in bases) {
+    // @ts-ignore
+    const multibase = bases[base];
+    if (multibase.prefix === prefix) {
+      return multibase;
+    }
+  }
+  // handle for CIDv0
+  if (prefix === 'Q') {
+    return bases.base58btc;
+  }
```

Comment:
This should *probably* be higher, but I like the idea that cidv0 will eventually be less common.. so i'm not moving this.